### PR TITLE
[READY] Makes PDA Messaging Visible

### PR DESCRIPTION
**Pull request by Nerev4r**
## About The Pull Request
Essentially, this plays a visible message (and a balloon alert) when someone sends a PDA message. A revival of https://github.com/Skyrat-SS13/Skyrat-tg/pull/10206 for the modern age. Think of it like how in some countries you can't mute the camera shutter noise on your phone.

## How This Contributes To The Skyrat Roleplay Experience
To quote my reformed orthodox rabbi Bill Clinton, 
CI policy allows for antagonists to immediately engage upon hearing any indication that you called for backup.

Clever players were shutting the ringer off on their PDAs and just PDAing security or coworkers whenever they ran into antags for backup so that the antagonist wouldn't know and wouldn't be able to act on them calling backup.

Also, this makes for better negative social interactions which creates finer RP. If you want to text your hoes on the side at a romantic dinner, you must be prepared to pay the price.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image
image
image
  
</details>

## Changelog
:cl:
add: PDAs now let nearby people know you're messaging someone.
/:cl:

### CODE REVIEW

**Comment by Nerev4r** [comment]
the extra 'you sent a message' has been cut

**Comment by RatFromTheJungle** [comment]
add a gun to this it should have a gun make the pda a gun

File: code/modules/modular_computers/file_system/programs/ntmessenger.dm [hunk 1]
```diff
@@ -283,6 +283,10 @@
 
 	if (prob(1))
 		message += " Sent from my PDA"
+	// SKYRAT EDIT BEGIN - PDA messages show a visible message; again!
+	user.visible_message(span_notice("[user]'s PDA rings out with the soft sound of keypresses."))
+	user.balloon_alert_to_viewers("sent a PDA message")
```
Comment (GoldenAlpharex) [inline code/modules/modular_computers/file_system/programs/ntmessenger.dm]: I feel like it should be one or the other, but not both.

Also the argument about it being basically like talking in a radio makes sense too, I feel like that's something that should have a range of like, one or two tiles tops.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Comment by Nerev4r** [comment]
image

ok

**Review by GoldenAlpharex — APPROVED** [review]
I really don't like this as a player, but I can't help but to agree on a conceptual level, as much as I hate people knowing that I'm typing on my PDA.

**Pull request merged**